goblin.go

```diff
@@ -99,16 +99,16 @@ func (d *Describe) run(g *G) bool {
 			a()
 		}
 
-		g.reporter.endDescribe()
+		g.reporter.EndDescribe()
 	}
 
 	return failed
 }
 
 type Failure struct {
-	stack    []string
-	testName string
-	message  string
+	Stack    []string
```

Comment:
These attributes can still be private don't they?

Reply:
no, they need to be exported. Otherwise `OtherReporter.Failure(f *Failure)` would error like this:

    f.message undefined (cannot refer to unexported field or method message)